packages/firebase/firestore/memory/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "firebase/firestore/memory",
```

Comment:
no license needed for json files.
Those fields are not needed in this file because they are already in the root package.json. You can provide a description for sure.

Reply:
Descriptions added to this package.json and top-level package.json.